src/Knp/Menu/Twig/MenuExtension.php

```diff
@@ -146,4 +171,30 @@ public function getName()
     {
         return 'knp_menu';
     }
+
+    /**
+     * @param ItemInterface $item
+     *
+     * @return ItemInterface|null
+     */
+    private function retrieveCurrentItem(ItemInterface $item)
```

Comment:
this should live in the Helper class instead, to be reusable by other templating engines